bus-mapping/src/operation.rs

```diff
@@ -113,13 +125,24 @@ impl Ord for MemoryOp {
 /// Represents a [`READ`](RW::READ)/[`WRITE`](RW::WRITE) into the stack implied
 /// by an specific [`OpcodeId`](crate::evm::opcodes::ids::OpcodeId) of the
 /// [`ExecStep`](crate::circuit_input_builder::ExecStep).
-#[derive(Debug, Clone, PartialEq, Eq)]
+#[derive(Clone, PartialEq, Eq)]
```

Comment:
Can't it be `Copy`?

Reply:
I don't see why MemoryOp, StackOp or StorageOp should be `Copy`.  I think the only types that should be copy are the ones that are treated like primitive types with a single small value (like Hash, Word, BlockNumber) or references.

Also, I don't think these types are cloned much, they are generated and moved to vectors.